sql/2020/09_Performance/lh6_vs_lh5_performance_score_02.sql

```diff
@@ -0,0 +1,24 @@
+#standardSQL
+
+# Calculates number of sites where the performance score changed low ( < 10), medium (10-30) or big (> 30) between
```

Comment:
Yup, this was mentioned in outline, but not in the "Metrics" section of the doc, so I wasn't sure if it's finalised or not. Let me add it to checklist and I'm going to work on them next. 

This one is more for "Changes to Performance Score from LH5 to LH6"

Reply:
What's the resolution here?

cc @max-ostapenko @thefoxis

Reply:
By distribution I meant what @rviscomi has outlined above; so how many sites (in %) are categorised as fast, slow and moderate. I believe that's what this phrasing referred to in the previous years' chapters, since that's where I got it from. 👍

Reply:
Ah, it's all new for me. Going to update this one.

Reply:
I see @dooman87 has already updated it to percentages.